src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,269 @@
+          setWorkflow(wf)
+        }
+      })
+
+    initialize()
+  }, [workflow]) // eslint-disable-line react-hooks/exhaustive-deps
+
+  /*
+   * Page render
+   */
+  const {
+    end,
+    failures,
+    start,
+    status,
+    workflowLog,
+    workflowName,
+    submittedFiles: { workflow: wdl } = {}
+  } = workflow
+
+  const restructureFailures = _.map(({ message, causedBy }) => ({
+    message,
+    ...(!_.isEmpty(causedBy) ? { causedBy: simplifyDidNotStartFailures(restructureFailures(causedBy)) } : {})
+  }))
+
+  const simplifyDidNotStartFailures = failuresArray => {
+    const filtered = _.filter(({ message }) => !_.isEmpty(message) && !message.startsWith('Will not start job'), failuresArray)
+    const sizeDiff = !_.isEmpty(failuresArray) ? failuresArray.length - filtered.length : 0
+    const newMessage = sizeDiff > 0 ? [{ message: `${sizeDiff} jobs were queued in Cromwell but never sent to the cloud backend due to failures elsewhere in the workflow` }]: []
+
+    return [...filtered, ...newMessage]
+  }
+
+  return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
+    div({ style: { marginBottom: '1rem', display: 'flex', alignItems: 'center' } }, [
+      h(Link, {
+        href: Nav.getLink('workspace-job-history', { namespace, name }),
+        style: { alignSelf: 'flex-start', display: 'inline-flex', alignItems: 'center', padding: '0.5rem 0' }
+      }, [icon('arrowLeft', { style: { marginRight: '0.5rem' } }), 'Job History']),
+      span({ style: { whiteSpace: 'pre' } }, [' > ']),
+      h(Link, {
+        href: Nav.getLink('workspace-submission-details', { namespace, name, submissionId }),
+        style: { alignSelf: 'flex-start', display: 'flex', alignItems: 'center', padding: '0.5rem 0' }
+      }, [`Submission ${submissionId}`]),
+      span({ style: { whiteSpace: 'pre' } }, [' > ']),
+      span({ style: Style.elements.sectionHeader }, [`Workflow ${workflowId}`])
+    ]),
+    _.isEmpty(workflow) ? centeredSpinner() : div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+      makeSection('Workflow Status', [
+        div({ style: { lineHeight: '24px' } }, [makeStatusLine(style => statusIcon(status, style), status)])
+      ]),
+      makeSection('Workflow Timing', [
+        table({ style: { marginTop: '0.3rem', lineHeight: '20px' } }, [
+          tbody([
+            tr([td({ style: styles.sectionTableLabel }, ['Start:']), start ? td([Utils.makeCompleteDate(start)]) : 'N/A']),
+            tr([td({ style: styles.sectionTableLabel }, ['End:']), end ? td([Utils.makeCompleteDate(end)]) : 'N/A'])
+          ])
+        ])
+      ]),
+      makeSection('Links', [
+        div({ style: { display: 'flex' } }, [
+          h(Link, {
+            ...Utils.newTabLinkProps,
+            href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
+            style: { padding: '.6rem' },
```

Comment:
Consider the following:
```suggestion
            style: { marginRight: '1.2rem' },
```
This will make the margin area non-clickable (padding is inside the element, so it adds to clickable area), as well as avoiding offsetting the first link from the section title.

Reply:
Switching to padding caused some layout issues, so I've reverted:

![image](https://user-images.githubusercontent.com/13006282/100802279-2ffd4c00-33f7-11eb-8f4b-15b9e60db334.png)

Reply:
Cleaned up.